Sources/MockoloFramework/Utils/TypeParser.swift

```diff
@@ -510,6 +510,25 @@ public final class `Type` {
         return mutableArg
     }
 
+    /// Parse throws clause from suffix and return typed-throw's type.
+    ///
+    /// - Returns: if typed-throw is used, returns its concrete type, unless returns nil.
+    static func extractTypedThrow(
+        suffix: String
+    ) -> String? {
+        return suffix
+            .components(separatedBy: .whitespaces)
+            .compactMap { clause in
+                guard let prefixRange = clause.range(of: "\(String.throws)(") else {
+                    return nil
+                }
+                let endIndex = clause.dropLast().endIndex
+                return String(
+                    clause[prefixRange.upperBound..<endIndex]
+                )
+            }
+            .first
+    }
```

Comment:
I am not sure if splitting by `.whitespace` is suitable or not. but this code works in added test-cases.

As far as I investigated, 
- `any Error` can't be thrown type because it's not a concrete type. 
- `some Error` can't come here due to an issue related to performance (https://github.com/swiftlang/swift-evolution/blob/main/proposals/0413-typed-throws.md#opaque-thrown-error-types).

Reply:
`throws(any Error)` is entirely possible.

The Swift's syntax has become more than complicated enough to look these up from `String` alone.
More structured information should be received from the arguments.

Reply:
Thank you for your information.

> throws(any Error) is entirely possible.

I did not know `any Error` can become throws type. I need to modify the implementation.

> The Swift's syntax has become more than complicated enough to look these up from String alone.
More structured information should be received from the arguments.

Absolutely. I felt like `suffix` variable in `MethodModel` should not be `String` but more structured type.